backend/src/utils/graph_db_utils.py

```diff
@@ -12,16 +14,13 @@
 
 
 def test_connection():
-    logging.info("testing database connection...")
     connection_healthy = False
     try:
         driver.verify_connectivity()
-        logging.info("database connection verified")
```

Comment:
I would argue that we want some sort of logging to confirm the db connection is working

Reply:
If there is a failure, then it should go to the exception where we are logging that exception.
logger.exception(f"Database connection failed: {e}")